style/auth.css

```
@@ -0,0 +1,204 @@
+header {
+    display: flex;
+    justify-content: center;
+}
+
+.login-page {
+    margin: 230px auto 24px;
+}
+
+.sign-up-page {
+    margin: 60px auto 24px;
+}
+
+.logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 24px;
+}
+
+.logo-img {
+    height: 103px;
+}
+
+.logo-text {
+    height: 62px;
+}
+
+form {
+    width: 640px;
+    margin: 0 auto;
+}
+
+label {
+    display: inline-block;
+    width: 640px;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 26px;
+}
+
+input {
+    display: inline-block;
+    padding: 16px 24px;
+    margin: 16px auto 24px;
+    width: 640px;
+    border-radius: 12px;
+    background-color: var(--gray100);
+    font-weight: 400;
+    color: var(--gray800);
+}
+
+input::placeholder {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+    color: var(--gray400);
+}
+
+.password {
+    position: relative;
+}
+
+.password, .check-password {
+    position: relative;
+}
+
+.show-password {
+    width: 24px;
+}
+
+.password button, .check-password button {
+    position: absolute;
+    right: 20px;
+    top: 54px;
+    background-color: var(--gray100);
+}
+
+.login-button, .sign-up-button {
+    display: inline-block;
+    padding: 16px;
+    width: 640px;
+    border-radius: 40px;
+    background-color: var(--gray400);
+    font-weight: 600;
+    color: var(--gray100);
+}
+
+.quick-login {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 74px;
+    padding: 0 23px;
+    margin: 24px auto;
+    border-radius: 8px;
+    background-color: #E6F2FF;
+}
+
+.sns {
+    display: flex;
+    align-items: center;
+    gap: 12px;
+}
+
+.sns img {
+    width: 42px;
+}
+
+.new-member {
+    display: flex;
+    justify-content: center;
+    gap: 4px;
+    text-align: center;
+    font-size: 14px;
+}
+
+.already-member {
+    display: flex;
+    justify-content: center;
+    gap: 4px;
+    text-align: center;
+    font-size: 14px;
+}
+
+.sign-up {
+    text-decoration: underline;
+    color: var(--blue);
+}
+
+.login {
+    text-decoration: underline;
+    color: var(--blue);
+}
+
+.login-button:hover {
+    background-color: var(--blue);
+}
+
+.sign-up-button:hover {
+    background-color: var(--blue);
+}
+
+@media (max-width: 1199px) {
+    .login-page {
+        margin: 190px auto 24px;
+    }
+    
+    .sign-up-page {
+        margin: 48px auto 24px;
+    }
+}
+
+@media (max-width: 767px) {
+    .login-page {
+        margin: 80px auto 24px;
+    }
+    
+    .sign-up-page {
+        margin: 24px auto 24px;
+    }
+
+    form {
+        max-width: 400px;
+        width: 100%;
+        padding: 16px;
+    }
```

💊 제안
`Mobile 사이즈에서 좌우 여백 16px 제외하고 내부 요소들이 너비를 모두 차지합니다.`, `Mobile 사이즈에서 내부 요소들의 너비는 기기의 너비가 커지는 만큼 커지지만 400px을 넘지 않습니다.`를 구현해주신 것 같아요~
저는 1번째 요구사항의 여백이라고 말하고 있기 때문에 form 영역이 아닌 margin 값이 더 적절할 것 같아요~
지금 구조에서는 width 값을 최대 너비 + 좌우 padding 값을 더해 주는게 변경이 제일 적을 것 같네요!

```suggestion
    form {
        max-width: 432px;
        width: 100%;
        padding: 16px;
    }
```